File: BAEKJOON/typescript/10000/10100/10156.ts
const fs = require("fs")
const filepath: string = process.platform === "linux" ? "/dev/stdin" : __dirname + "/input.txt"
const input: number[] = fs.readFileSync(filepath).toString().trim().split(" ").map((x: string) => parseInt(x))

const [K, N, M] = input
const tmp: number = K * N - M

if (tmp < 0) {
  console.log(0)
} else {
  console.log(tmp)
}

File: BAEKJOON/Golang/10000/10100/10156.go
package main

import "fmt"

func main() {
	var K, N, M int
	fmt.Scanf("%d %d %d", &K, &N, &M)
	tmp := K*N - M
	if tmp < 0 {
		fmt.Println(0)
	} else {
		fmt.Println(tmp)
	}
}
